buildscripts/rethinkdbintegrationtest.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
After all the complexity via code repetition in the docker build scripts that sounds like a really good idea.